ts/features/bonus/cgn/api/backendCgnMerchants.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  ApiHeaderJson,
+  composeHeaderProducers,
+  createFetchRequestForApi,
+  RequestHeaderProducer,
+  RequestHeaders
+} from "italia-ts-commons/lib/requests";
+import { Omit } from "italia-ts-commons/lib/types";
+import { defaultRetryingFetch } from "../../../../utils/fetch";
+import {
+  getMerchantDefaultDecoder,
+  GetMerchantT,
+  getOfflineMerchantsDefaultDecoder,
+  GetOfflineMerchantsT,
+  getOnlineMerchantsDefaultDecoder,
+  GetOnlineMerchantsT
+} from "../../../../../definitions/cgn/merchants/requestTypes";
+
+const tokenHeaderProducer = ParamAuthorizationBearerHeaderProducer();
```

Comment:
You could use the util [tokenHeaderProducer](https://github.com/pagopa/io-app/blob/add867d181e88582f2c166bd0d785d858280572d/ts/utils/api.ts#L27) function